stellar-captive-core-api/README.md

```diff
@@ -0,0 +1,11 @@
+# Stellar Captive Core API
+The Captive Stellar-Core Server allows you to run a dedicated Stellar-Core instance for the purpose of ingestion. The server must be bundled with a Stellar Core binary.
```

Comment:
`The server must be bundled with a Stellar Core binary` <-- is this something users have to do? Or it's just FYI that the binary is included in this package?

Reply:
The package defines `stellar-core` as a dependency, so this is more an explanation as to how the HTTP API sits in an architecture

Reply:
I understand "bundled with" as "included". If it's just a dependency maybe say so?